### PR TITLE
Add tags to package.json

### PR DESCRIPTION
**Pull request by SanderElias**
Created a PR for this, so we can discuss the tags/keywords before merging

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,16 @@
...
   },
+  "tags": [
+    "url",
+    "urlpattern",
+    "url-pattern",
+    "type-script",
+    "javascript",
+    "browser",
+    "node",
+    "polyfill"
```
Comment (kenchris) [inline package.json]: Maybe "w3c", "wicg", "web" as well?

Is there anywhere we can see common tags?

**Review by kenchris — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -14,6 +14,16 @@
     "import": "./dist/index.js",
     "require": "./dist/index.cjs"
   },
+  "tags": [
+    "url",
+    "urlpattern",
+    "url-pattern",
+    "type-script",
```
Comment (SanderElias) [inline package.json:21]:
```suggestion
```
Perhaps it is better to drop those?

Comment (kenchris) [inline package.json]: why not typescript as one word?

Comment (SanderElias) [reply package.json]: I think its better to drop it completely in hind-sight.

File: package.json [hunk 3]
```diff
@@ -14,6 +14,16 @@
...
+    "javascript",
+    "browser",
+    "node",
```
Comment (SanderElias) [inline package.json]:
```suggestion
```
Remove those as they don't add any real value

File: package.json (hunk 1 above)
Comment (kenchris) [inline package.json]:
```suggestion
    "polyfill",
    "w3c",
    "wicg"
```

File: package.json (hunk 2 above)
Comment (kenchris) [inline package.json]:
```suggestion
```

File: package.json [hunk 4]
```diff
@@ -14,6 +14,16 @@
     "import": "./dist/index.js",
     "require": "./dist/index.cjs"
   },
+  "tags": [
+    "url",
+    "urlpattern",
+    "url-pattern",
+    "type-script",
+    "javascript",
```
Comment (kenchris) [inline package.json]:
```suggestion
```

**Review by kenchris — APPROVED** [review]

**Pull request merged**